mix.exs

```diff
@@ -26,7 +26,7 @@ defmodule PublicSuffix.Mixfile do
 
   defp deps do
     [
-      {:idna, ">= 1.2.0 and < 3.0.0"},
+      {:idna, "~> 5.0"},
```

Comment:
This requires that every user of this library upgrade idna to 5.x, but that might be difficult if they have other dependencies that rely on earlier versions.  Can you change this to `">= 1.2.0 and < 6.0.0"` so that 5.x versions are supported but not required?